lib/tasks/TaskUtil.js

```diff
@@ -68,14 +68,14 @@ class TaskUtil {
 	 * This method is only available to custom task extensions defining
 	 * <b>Specification Version 2.2 and above</b>.
 	 *
-	 * @param {module:@ui5/fs.Resource} resource The resource the tag should be retrieved for
+	 * @param {string|module:@ui5/fs.Resource} resourcePath Path or resource-instance the tag should be retrieved for
 	 * @param {string} tag Name of the tag
 	 * @returns {string|boolean|integer|undefined} Tag value for the given resource.
 	 * 										<code>undefined</code> if no value is available
 	 * @public
 	 */
-	getTag(resource, tag) {
-		return this._projectBuildContext.getResourceTagCollection().getTag(resource, tag);
+	getTag(resourcePath, tag) {
+		return this._projectBuildContext.getResourceTagCollection().getTag(resourcePath, tag);
```

Comment:
```suggestion
		return this.getResourceTagCollection().getTag(resourcePath, tag);
```

Reply:
Removed the (unused) getResourceTagCollection function